etc/cdn-log-shipper/log-shipper.yml

```diff
@@ -171,6 +171,15 @@ Resources:
             });
             fields.push('prx-listener-id');
 
+            datas.forEach(data => {
+              if (data['sc-range-start'].match(/^[0-9]+$/)) {
+                data['byte-range'] = `${data['sc-range-start']}-${data['sc-range-end']}`;
+              } else {
+                data['byte-range'] = '-';
```

Comment:
Would we ever hit this?

Reply:
Yes, it only has numeric values for a 206 response, otherwise start and end are dashes

Reply:
Oh right - this is the actual request Range header field.  Vs the "byte range the server sent to the client" we use for IAB2 calculations.